src/main/java/step2/CalculateOperator.java

```diff
@@ -0,0 +1,35 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum CalculateOperator {
+  PLUS("+", (x, y) -> x + y),
+  MINUS("-", (x, y) -> x - y),
+  MULTIPLY("*", (x, y) -> x * y),
+  DIVIDE("/", (x, y) -> {
+    if (y == 0) {
+      throw new IllegalArgumentException();
+    }
+    return x / y;
+  });
+
+  private String operator;
+  private BiFunction<Integer, Integer, Integer> expression;
+
+  CalculateOperator(String operator, BiFunction<Integer, Integer, Integer> expression) {
+    this.operator = operator;
+    this.expression = expression;
+  }
+
+  public static int calculate(String operator, Integer x, Integer y) {
```

Comment:
```suggestion
  public static int calculate(String operator, int x, int y) {
```
primitive type 사용을 추천합니다
Integer 타입을 쓰면 null을 전달 할 수 있는 여지를 남기게 됩니다.
그렇게 되면 null 방어로직을 추가해야되니 아예 숫자만 받게 만드는게 좋겠네요
외부에서 값을 받아올 때 null까지 포함해서 생각해야되는 경우도 있지만 지금 과제에서는 null은 안 받도록 처리하는게 적절해보이네요